containers/squid.sh

```diff
@@ -11,9 +11,18 @@ set -eu
 MYDIR=$(dirname $(realpath "$0"))
 CRUN=${CRUN:-$(which podman docker 2>/dev/null | head -n1)}
 
-if [ "${1:-}" = start ]; then
+is_running() {
+    [ -n "$($CRUN ps -q -f 'name=^squid$')" ]
+}
+
+start() {
+    if is_running; then
+        echo "Already running"
+        return 0
```

Comment:
Wouldn't it be better to use ``exit 0`` instead to avoid issues when the main part of this script is extended.

Reply:
No, I don't think so. The idea is that start() makes sure that the container is started, and that's the case if it's already running. If there is some subsequent action, then that should run no matter whether the container actually got started IMHO.